gradle/maven.gradle

```diff
@@ -39,7 +39,7 @@ plugins.withId('maven') {
                     }
                 }
 
-                pom.artifactId = project.path.replace(':', '')
+                pom.artifactId = project.path.replaceFirst(':', '').replace(':', '-')
                 pom.project {
                     name "Pravega"
                     url "http://pravega.io"
```

Comment:
The Pravega website is now `https://...`. :)

Also, `licenses` and `developers` missing.

Reply:
that is an unrelated change. Have opened a separate issue to tackle it (#91)